lms/envs/common.py

```diff
@@ -590,6 +590,18 @@
     }
 }
 
+COURSE_MODE_DEFAULTS = {
+    'bulk_sku': None,
```

Comment:
ah got it thanks. those commerce values must be what power the about page details for the open edx about pages. thanks!